test/python/transpiler/test_vf2_layout.py

```diff
@@ -293,9 +293,12 @@ class TestVF2LayoutLattice(LayoutTestCase):
     cmap25 = CouplingMap.from_hexagonal_lattice(25, 25, bidirectional=False)
 
     def graph_state_from_pygraph(self, graph):
-        """Creates a GraphState circuit from a PyGraph"""
+        """Creates a GraphStateGate circuit from a PyGraph"""
         adjacency_matrix = rustworkx.adjacency_matrix(graph)
-        return GraphState(adjacency_matrix).decompose()
+        num_qubits = adjacency_matrix.shape[0]
+        qc = QuantumCircuit(num_qubits)
+        qc.append(GraphStateGate(adjacency_matrix), range(num_qubits))
+        return qc.decompose()
```

Comment:
How about the following?
```suggestion
        return GraphStateGate(adjacency_matrix).definition
```